### DISCUSSION TITLE
How to properly install a second instance of Restreamer?

### DISCUSSION THREAD

**Discussion by MauricioOnetto** [datarhei/restreamer#450] [Q&A] [answered]
Hi
I'm using Restreamer (behind NGINX) for a while and it works smooth
Thanks for that

Now I'm trying to install a second instance of Restreamer (same server) but I'm missing something

I could install it and validate with a new user (and new subdomain), but when I try to connect with OBS using the wizard, It doesnt detect any transmision (and I'm sending from OBS to the wizard's rtmp url)

This is my code to install
docker run -d --restart=always --name restreamer11 -v /opt11/restreamer/config:/core/config -v /opt11/restreamer/data:/core/data -p 8011:8080 -p 8111:8181 -p 2011:1935 -p 2111:1936 -p 6011:6000/udp datarhei/restreamer:latest

I'm using UFW (any new port is allowed)
8011/tcp
8111/tcp
2011/tcp
2111/tcp
6011/udp

What am I missing?

Thank you

**Comment by svenerbeck** [comment, marked as answer]
Hi @MauricioOnetto
I took the liberty of making an issue out of your request. https://github.com/datarhei/restreamer/issues/464
Cheerio
Sven
